collect_app/src/main/res/layout/drive_layout.xml

```diff
@@ -16,94 +15,96 @@ the License.
 
 
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                android:layout_width="fill_parent"
-                android:layout_height="fill_parent"
-                android:layout_alignParentTop="true" >
+    android:layout_width="fill_parent"
```

Comment:
For what it’s worth, I like to do an initial commit with just formatting changes, followed by a commit with the substantive changes, so the latter aren’t buried in the noise of the former.